src/Compilers/VisualBasic/Test/Syntax/Parser/ParseExpression.vb

```diff
@@ -445,13 +445,7 @@ ToString]]>.Value)
                     Dim y = New (A As Integer, B$)
                 End Sub
             End Module
-        ]]>,
-<errors>
-    <error id="37280" message="'New' cannot be used with tuple type. Use a tuple literal expression instead." start="87" end="93"/>
-    <error id="37280" message="'New' cannot be used with tuple type. Use a tuple literal expression instead." start="126" End="132"/>
-    <error id="37280" message="'New' cannot be used with tuple type. Use a tuple literal expression instead." start="167" End="185"/>
-</errors>
-        )
+        ]]>)
```

Comment:
:memo: I added a corresponding test to check binding diagnostics in `CodeGenTuples.vb`